src/MaxText/utils/ckpt_conversion/to_maxtext.py

```diff
@@ -182,6 +182,18 @@ def _build_single_axis_stacked_tensor(
   return np.stack(tensors_to_stack, axis=axis_to_stack)
 
 
+def _get_hf_model(model_id: str, token: str):
```

Comment:
I like this design. I recall that Llama4 multimodal is also not supported in AutoModelForCausalLM and have a separate "Llama4ForConditionalGeneration" module. Have we supported Llama4 in this ckpt conversion script?